### PR TITLE
add xvg support

### PR DESCRIPTION
**Pull request by justinvforvendetta**

### CODE REVIEW

**Comment by justinvforvendetta** [comment]
this wont work until nbitcoin is updated =]

File: NBXplorer.Client/NBXplorerNetworkProvider.Verge.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    public partial class NBXplorerNetworkProvider
+    {
+		private void InitVerge(NetworkType networkType)
+		{
+			Add(new NBXplorerNetwork(NBitcoin.Altcoins.Verge.Instance, networkType)
+			{
+				MinRPCVersion = 6000200,
+				ChainLoadingTimeout = TimeSpan.FromHours(1),
+				ChainCacheLoadingTimeout = TimeSpan.FromMinutes(3),
+				CoinType = NetworkType == NetworkType.Mainnet ? new KeyPath("0'") : new KeyPath("1'")
```
Comment (NicolasDorier) [inline NBXplorer.Client/NBXplorerNetworkProvider.Verge.cs]: incorrect keypath for xvg

Comment (justinvforvendetta) [reply NBXplorer.Client/NBXplorerNetworkProvider.Verge.cs]: @NicolasDorier ah yes, thanks. fixed!

**Comment by NicolasDorier** [comment]
test fails

**Comment by lepetitpois** [comment]
Info about XVG support highly appreciated. Any news on progess?

**Pull request closed without merging**